exercises/practice/react/build.gradle

```diff
@@ -0,0 +1,30 @@
+apply plugin: "java"
+apply plugin: "eclipse"
+apply plugin: "idea"
+
+// set default encoding to UTF-8
+compileJava.options.encoding = "UTF-8"
+compileTestJava.options.encoding = "UTF-8"
+
+repositories {
+  mavenCentral()
+}
+
+gradle.projectsEvaluated {
+  tasks.withType(JavaCompile) {
+    options.compilerArgs << "-Xlint:unchecked"
+  }
+}
```

Comment:
Any particular reason to add this setting for this exercise? Other exercises don't seem to need it.

Reply:
No, I can't remember why I added it in the first place ... I'll remove it.